app/controllers/users_controller.rb

```diff
@@ -49,6 +52,17 @@ def enroll
   end
 
   ####################################################
+  # User listing page for Admins                     #
+  ####################################################
+  def index
+    @users = if params[:term].present?
```

Comment:
Calling this parameter `term` is confusing, in part because it's vague and in part because the Course model has a `term` attribute that means something different. In the current implementation, it's specifically an email search, so maybe call it `email`?